packages/cluster-ui/src/table/table.module.scss

```diff
@@ -1,26 +1,31 @@
 @import "src/core/index.module";
+@import "src/sortedtable/tableHead/tableHead.module";
 
 .crl-table-wrapper {
-  .ant-table {
+  :global(.ant-table) {
```

Comment:
so the idea here is that we are allowing the application to define global styles?

Reply:
It just tells to treat ".ant-table" class as a global one because it's defined outside of this CSS module. 
`:global` annotation prevents CSS modules to change the name of this class so this selector can work with global Antd styles.

Reply:
I see. I don't love the way this looks, but I don't love using the Ant Table, and it looks like it works. 😄